spec/unit/module/git_spec.rb

```diff
@@ -128,6 +128,25 @@
       subject.sync
       expect(Dir.exist?(spec_path)).to eq true
     end
+
+    it 'returns true if repo was updated' do
+      allow(mock_repo).to receive(:resolve).with('master').and_return('abc123')
+      expect(mock_repo).to receive(:sync).and_return(true)
+      expect(subject.sync).to be true
+    end
+
+    it 'returns false if repo was not updated (in-sync)' do
+      allow(mock_repo).to receive(:resolve).with('master').and_return('abc123')
+      expect(mock_repo).to receive(:sync).and_return(false)
+      expect(subject.sync).to be false
+    end
+
+    it 'returns false if `should_sync?` is false' do
+      # modules do not sync if they are not requested
+      mod = described_class.new(title, dirname, { overrides: { modules: { requested_modules: ['other_mod'] } } })
+      allow(mock_repo).to receive(:resolve).with('master').and_return('abc123')
```

Comment:
Should this get pulled into a `before` block, since we use it in all the tests within this `describe`?